### PR TITLE
small correction re min. number of nodes

### PR DESCRIPTION
**Pull request by cjschroder**
for ceilometer cluster

see https://bugzilla.suse.com/show_bug.cgi?id=1032692

### CODE REVIEW

**Comment by cjschroder** [comment]
MongoDB reference: https://docs.mongodb.com/v3.0/core/sharded-cluster-architectures-production/

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -4136,9 +4136,7 @@ getent group postgres | cut -d: -f3</screen>
     Making &o_meter; highly available requires no special
     configuration&mdash;it is sufficient to deploy the roles
     <guimenu>ceilometer-server</guimenu> and
-    <guimenu>ceilometer-polling</guimenu> on a cluster. The cluster needs to
-    consist of an odd number of nodes, otherwise the &o_meter; deployment will
-    fail.
+    <guimenu>ceilometer-polling</guimenu> on a cluster. If you are using MongoDB for your backend the cluster must have a minimum of three nodes. If you are using MySQL or PostgreSQL, you may use two nodes. (MongoDB is the recommended backend for best performance.)
```
Comment (taroth21) [inline xml/depl_nodes.xml]: backend -> back-end (according to Style Guide)

Comment (taroth21) [inline xml/depl_nodes.xml]: And could you please mention this change in the docupdates, too? (see the empty skeleton in depl_docupdates.xml, sec.deploy.docupdates.c7_maint2) Thanks.

Comment (cjschroder) [inline xml/depl_nodes.xml]: Thanks, I'll open a separate PR for depl_docupdates.xml.

**Comment by taroth21** [comment]
Thanks!

**Pull request merged**